Ac 1409 prepare release notes 202212.0 for publishing (#1457)
spryker/spryker-docs: docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202212.0/release-notes-202212.0.md

Spryker has recently extended the range of functionalities for Configurable Products. Customers can now save configurable products in their shopping list and wishlist, where they can also finalize or change the configuration of these products.

Configurable Products are useful for items with a range of customizable options. For example, when buying a t-shirt from your online store, customers can choose the size, material, and color or even add their name or logo to the product. If you are selling services, Configurable Products let customers select a preferred time and date for the delivery of that service.

Customers can access a standalone configurator page and choose from various available product options based on their preferences. Depending on the configuration choices, the product price is adjusted accordingly.

![configurable-product-on-the-storefront](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Configurable+Product/Configurable+Product+feature+overview/configurable-product-on-the-storefront.gif)

> Configured Products can be saved, accessed, and configured again in carts or from the product details page. Customers can also easily reorder past purchases. Configurable Products can also be part of a [Request For Quote (RFQ)](/docs/pbc/all/request-for-quote/request-for-quote.html)—the Spryker's powerful Quotation and Management feature can overwrite all the automatically calculated prices from the Configurable Product tool.

Reviewer: ```suggestion
Configured Products can be saved, accessed, and configured again in carts or from the product details page. Customers can also easily reorder past purchases. Configurable Products can also be part of a [Request For Quote (RFQ)](/docs/pbc/all/request-for-quote/request-for-quote.html) — Spryker's powerful Quotation and Management feature can overwrite all the automatically calculated prices from the Configurable Product tool.
```

**Business benefit**: Configurable Products maximize customer satisfaction through product personalization and by providing a robust customer self-service experience. This also adds value to your business by reducing the lead times Sales teams would previously have to dedicate to finalizing product configurations with customers.

#### Documentation

* Technical prerequisites: [Product Configuration feature integration](/docs/scos/dev/feature-integration-guides/202212.0/product-configuration-feature-integration.html)
* [Configurable Product feature overview](/docs/scos/user/features/202204.0/configurable-product-feature-overview.html)

### Localization improvements <span class="inline-img">![improvement](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/releases/release-notes/improvement.png)</span>

> Adapting your offerings to your customers' needs and customs is crucial for providing a superior user experience and ultimately boosting conversion rates. To facilitate this, we've improved the number formatting for any locale. Based on the locale and browser language setting, your customers in the Storefront and your company users in the Back Office are shown the number format they expect to see. Look at the following example with a product price:

Reviewer: ```suggestion
Adapting your offerings to your customers' needs and locales is crucial for providing a superior user experience and ultimately boosting conversion rates. To facilitate this, we've improved the number formatting for any locale. Based on the locale and browser language setting, your customers in the Storefront and your company users in the Back Office are shown the number format they expect to see. Look at the following example with a product price:
```

| LOCALE | PRODUCT PRICE |
|---|---|
| de_DE | 123.456,78 |
| en_US | 123,456.78 |

#### Number formatting in the Storefront <span class="inline-img">![improvement](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/releases/release-notes/improvement.png)</span>

For an effortless shopping experience, the localized number formatting is available throughout the Storefront for any user locale. Whether it is a catalog, product details page, cart, checkout, or customer account, your customers and guests can now enjoy their known number format. This helps to avoid any confusion about pricing or quantities.

##### Documentation

[HowTo: Add support for number formatting in the Storefront](/docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/howto-add-support-for-number-formatting-in-the-storefront.html#prerequisites)

#### Number formatting in the Back Office <span class="inline-img">![improvement](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/releases/release-notes/improvement.png)</span>

> Efficiency and accuracy are essential when working in the Back Office. Letting you increase the efficiency of your company users in any locale, locale-based number formats are available in all relevant places throughout the Back Office. Whether it's prices, quantities, or stocks: Your company users can stay focused on their tasks while enjoying the number format they are familiar with.

Reviewer: ```suggestion
Efficiency and accuracy are essential when working in the Back Office. Increasing the efficiency of your company users in any locale, locale-based number formats are available in all relevant places throughout the Back Office. Whether it's prices, quantities, or stocks: your company users can stay focused on their tasks while enjoying the number format they are familiar with.
```
